doc/src/site/sphinx/troubleshooting.rst

```diff
@@ -0,0 +1,101 @@
+Production Checklist & Troubleshooting Guide
+********************************************
+
+Production Checklist
+====================
+
+In this section we provide you the production environment basic settings.
+
+First of all, you should check the different systems and services availability in your environment.
+
+Then to configure our service you need to take a look at two files.
+
+application.conf
+----------------
+Where? -> /etc/sds/sparkta/application.conf
+
+What should you check?
+
+#. Network interfaces and ports
+#. Zookeeper configuration
+#. Hdfs paths
+#. Resource manager configuration (mesos/yarn/standalone/local)
+#. Spark configurations
+
+run.sh
+------
+Where? -> /opt/sds/sparkta/bin/run.sh
+
+What should you check?
+
+#. Memory configuration
+#. Jmx port
+#. Other Java opts
+
+Finally you should check your inputs and outputs availability and their configuration in your policies.
+
+
+Troubleshooting
+===============
+
+This section tries to group the common errors our users find. Most of them are easily fixed checking the logs.
+Sparta logs are at `/var/log/sds/sparkta/sparkta.[out|err]` . Please if you find an error that belongs to Sparkta don't
```

Comment:
It is more polite do not use contractions in documentation. I prefer "do not" than "don't"